README.md

```diff
@@ -7,3 +7,5 @@ Look no further.
 
 The types in this package are not special cased by type checkers in any way. Feel free to vendor
 them in your project instead of depending on this package.
+
+To use these types, you need at least mypy 1.4 or pyright 1.1.314.
```

Comment:
Is that because of the Buffer stuff? I feel most of the library should work on much older versions.

Reply:
Yes, because of Buffer

Reply:
Then I feel this note is too strongly worded. Maybe:

```suggestion
While most of this library will work on any recent type checker version,
some types depend on newer type checker features. We guarantee
support on mypy 1.4 and pyright 1.1.314 or higher.
```

Reply:
Also this possibly implies that this library won't with at all with pyre/pytype, which feels like something we don't necessarily want to say

Reply:
Or pyanalyze :D. I think the general principle should be that we test and support recent mypy and pyright, and support others on a best-effort basis. Which is basically what typeshed does too.